### ISSUE TITLE
Python version conflicts in README and requirement.txt

### ISSUE THREAD

**Issue by Ynjxsjmh** [Shiyang-Yan/Discrete-continous-PG-for-Retrieval#3]
Python version lists in your [requirement.txt](https://github.com/Shiyang-Yan/Discrete-continous-PG-for-Retrieval/blob/58c910629f97ed98f8168fd9074b038e6de71e5b/requirement.txt) is 2.7.15 which is different from the version 3.8 listed in your README. Which one is the right Python version?

https://github.com/Shiyang-Yan/Discrete-continous-PG-for-Retrieval/blob/58c910629f97ed98f8168fd9074b038e6de71e5b/requirement.txt#L17

https://github.com/Shiyang-Yan/Discrete-continous-PG-for-Retrieval/blob/58c910629f97ed98f8168fd9074b038e6de71e5b/README.md?plain=1#L13

**Comment by Shiyang-Yan** [comment]
Yes, there is a conflict between the requirements.txt and README. We recommend Python 3.8 for this project. Please ignore the version in requirements.txt but the packages are truly required in the project. You could just pip install the latest version, which will work.

**Comment by Ynjxsjmh** [comment]
Thanks for your reply and sorry for the late reply due to the client mail received error caused by Google forcing two-step verification. The model runs fine without the version constraint.

There are two packages [`opts`](https://github.com/KunpengLi1994/VSRN/blob/master/opts.py) and `torchnlp` that seem useless to the model, do they matter much?

https://github.com/Shiyang-Yan/Discrete-continous-PG-for-Retrieval/blob/39fd7a81f732ae043c2ea20352a0c55b72834639/model_PG.py#L16

https://github.com/Shiyang-Yan/Discrete-continous-PG-for-Retrieval/blob/39fd7a81f732ae043c2ea20352a0c55b72834639/model_PG.py#L662-L663

**Comment by Shiyang-Yan** [comment]
No, opts seems useless but the torchnlp is useful. We used the GloVe to encode the language. Sorry that the code has not been organized properly. But using it is okay with some necessary debugging.

**Comment by Ynjxsjmh** [comment]
Thanks for your reply.

> We used the GloVe to encode the language. 

Could you explain this more. I try to search `GloVe` in this repo, it gives 3 results. However, none of them is related with code usage.

![image](https://user-images.githubusercontent.com/40143136/148203826-0437df6b-e2f5-43a2-8a8f-2edfe0f8ff91.png)
